src/main/java/edu/greenblitz/robotName/commands/arm/roller/RollerCommand.java

```diff
@@ -0,0 +1,13 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.Roller;
+import edu.greenblitz.robotName.utils.GBCommand;
+
+public abstract class RollerCommand extends GBCommand {
```

Comment:
suggestion:
add a command run in and run out with depending on the beam breaker. the run and reverse run commands should only run the motor and another command should decide if to stop it.

Reply:
It is not only a suggestion, it is a comment now